llpc/util/llpcElfWriter.h

```diff
@@ -74,6 +74,22 @@ template <class Elf> class ElfWriter {
 
   static void updateMetaNote(Context *context, const ElfNote *note, ElfNote *newNote);
 
+  static size_t getRelocsCount(const SectionBuffer *pRelocSection);
+
+  static size_t getRelocPsStartPos(const SectionBuffer *pRelocSection, size_t psIsaOffset);
+
+  void createNewSection(const char *pSectionName, SectionBuffer *pInputSection, SectionBuffer *pNewSection);
+
+  void mergeRelocSection(const ElfReader<Elf> &reader, const SectionBuffer *pSection1, size_t section1RelocsCount,
```

Comment:
Could this calculate the number of relocs internally, instead of having this as arguments?
Also, could the new section pointer be the return value?

Reply:
To calculate the available number of the relocs, it still needs to pass the `PsIsaOffset `.